src/scss/components/_newGameButton.scss

```diff
@@ -1,6 +1,5 @@
 .newGameButton {
   display: flex;
-  flex-direction: row-reverse;
   justify-content: center;
   align-items: center;
   padding-right: 50px;
```

Comment:
Te wszystkie właściwości podzieliłbym na grupy (można je oddzielić spacjami),
np. czcionki osobno, kolory osobno, pozycjonowanie osobno. 
Bo jak tak się patrzy na to, to ściana kodu.

Reply:
Hmm nie wiem czy to dobra praktyka, wtedy by tak w każdej klasie trzeba było robić, można spytać Piotra później ale wydaje mi się, że powinno zostać tak jak jest.

Reply:
Tak, powinniśmy tak robić ;)
Nie wprowadzam jeszcze wszystkiego na raz, ale tak. Nie oddzielamy enterami, ale z założenia grupujemy.
Pokażę Wam to na szkoleniu w następnym tygodniu.